src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,55 @@
+package org.example.service;
+
+import java.sql.*;
+
+
+public class ConnectionService {
+    private static final String DATABASE_NAME = "postgres";
+    private static final String DATABASE_USER = "postgres";
+    private static final String DATABASE_PASSWORD = "postgres";
+    private static final String DRIVER_POSTGRESQL = "org.postgresql.Driver";
+    private static final String SCHEMA = "jdbc:postgresql";
+    private static final String HOST = "developers-db";
+    private static final String PORT = "5432";
+    private final Connection connection;
+    private static ConnectionService INSTANCE;
```

Comment:
Check the naming conventions.
When the variable title should be written in caps.

Reply:
fixed